ALERT: Assert failure in a(1) >= 20 && a(1) <= 28 at examples/release/call_cfgfail.c:13:1, with domain
a(1): ⊤
b(2): ⊤
a(4): ⊤

NODE 1 [examples/release/call_cfgfail.c:6:0]: (took 6 iterations) [ENTRY POINT: add]
a(1): ⊤
b(2): ⊤
a(4): 0

NODE 2 [examples/release/call_cfgfail.c:8:1]: (took 5 iterations) [EXIT POINT: add]
a(1): ⊤
b(2): ⊤
return value (3): ⊤
a(4): 0

NODE 3 [examples/release/call_cfgfail.c:10:0]: (took 1 iterations) [ENTRY POINT: main]
a(1): 0
b(2): 0
a(4): 0

NODE 4 [examples/release/call_cfgfail.c:14:1]: (took 2 iterations) [EXIT POINT: main]
a(1): ⊤
b(2): ⊤
a(4): ⊤

NODE 5 [examples/release/call_cfgfail.c:12:1]: (took 2 iterations)
a(1): ⊤
b(2): ⊤
a(4): ⊤

NODE 6 [examples/release/call_cfgfail.c:11:16]: (took 2 iterations)
a(1): 0
b(2): 3
a(4): 0

NODE 7 [examples/release/call_cfgfail.c:11:14]: (took 2 iterations)
a(1): 2
b(2): 3
a(4): 0

NODE 8 [examples/release/call_cfgfail.c:11:12]: (took 5 iterations)
a(1): ⊤
b(2): ⊤
a(4): 0

NODE 9 [examples/release/call_cfgfail.c:11:12]: (took 5 iterations)
a(1): ⊤
b(2): ⊤
a(4): 0

NODE 10 [examples/release/call_cfgfail.c:11:27]: (took 4 iterations)
a(1): ⊤
b(2): 4
a(4): 0

NODE 11 [examples/release/call_cfgfail.c:11:25]: (took 4 iterations)
a(1): 3
b(2): 4
a(4): 0

NODE 12 [examples/release/call_cfgfail.c:11:23]: (took 2 iterations)
a(1): ⊤
b(2): ⊤
a(4): 0

NODE 13 [examples/release/call_cfgfail.c:11:23]: (took 2 iterations)
a(1): ⊤
b(2): ⊤
a(4): 0

NODE 14 [examples/release/call_cfgfail.c:13:1]: (took 2 iterations)
a(1): ⊤
b(2): ⊤
a(4): ⊤

NODE 15 [examples/release/call_cfgfail.c:6:0]: (took 0 iterations)
a(1): 0
b(2): 0
a(4): 0

NODE 16 [examples/release/call_cfgfail.c:16:0]: (took 1 iterations)
a(1): 0
b(2): 0
a(4): 0

==============================
ALERT: 1 alert was raised! (See above)
==============================
